examples/pod_spec_path/README.md

```diff
@@ -0,0 +1,156 @@
+# Testing PodSpec path
+
+Based on the note by @qibobo 
+
+Create the application CRD:
+
+```
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: appconfigs.stable.example.com
+spec:
+  group: stable.example.com
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  scope: Namespaced
+  names:
+    plural: appconfigs
+    singular: appconfig
+    kind: AppConfig
+    shortNames:
+    - ac
+```
+
+Create the application CR using the CRD:
+
+```
+apiVersion: "stable.example.com/v1"
+kind: AppConfig
+metadata:
+  name: demo-appconfig
+spec:
+  uri: "some uri"
+  Command: "some command"
+  image: my-image
+  spec:
+    containers:
+    - name: hello-world
+      # Image from dockerhub, This is the import path for the Go binary to build and run.
+      image: yusufkaratoprak/kubernetes-gosample:latest
+      ports:
+      - containerPort: 8090
+```
+
+Create the operator source:
+
+```
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: db-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: pmacik
+```
+
+Install the Postgres Operator and then create Database CR:
+
+```
+apiVersion: postgresql.baiju.dev/v1alpha1
+kind: Database
+metadata:
+  name: db-demo
+spec:
+  image: docker.io/postgres
+  imageName: postgres
+  dbName: db-demo
+```
+
+Create the SBR with a custom bind path:
+
+```
+apiVersion: apps.openshift.io/v1alpha1
+kind: ServiceBindingRequest
+metadata:
+  name: binding-request-sample
+spec:
+  envVarPrefix: qiye111
+  applicationSelector:
+    resourceRef: demo-appconfig
+    group: stable.example.com
+    version: v1
+    resource: appconfigs 
+    bindingPath:
+      podSpecPath:
+        containers: spec.spec.containers
+        volumes: spec.spec.volumes
```

Comment:
this should have some effect to on application resource? I do not see `spec.spec.volumes` structure in the example under.

Reply:
This doc has been converted to an acceptance test.